src/index.js

```diff
@@ -36,7 +36,46 @@ import { getReleaseInfo } from "./util.js";
  */
 
 /**
- * Automates building an NW.js application.
+ * Installation Guide
+ *
+ * Every NW.js release includes a modified Node.js binary at a specific version. It is recommended to [install](https://nodejs.org/en/download/package-manager) a version greater than or equal to NW.js's Node version. Consult the [version manifest](https://nwjs.io/versions) on the version to install. Install using the package manager of choice. The recommended method is via [corepack](https://nodejs.org/api/corepack.html).
```

Comment:
You actually want the *exact* version that is built in. For example, if I was using NW.js 0.14.7 which comes with Node 5.11.1. If I'm on Node 19, then when I `npm install` a dependency it may download or build files compatible with Node's ABI based on the global install, which with 19 is 111 but with Node 5, the ABI is 47. So the files downloaded or built will not be compatible with the Node version built in to NW.js.

Reply:
Right makes sense. Thanks for catching this - would have definitely overlooked this!